base/src/org/compiere/model/MScheduler.java

```diff
@@ -223,11 +226,7 @@ public Integer[] getRecipientByUserIds()
 				}
 			}
 		}
-		//	Add Updater
-		if (list.size() == 0)
-		{
-			list.add(getCreatedBy());
-		}
+

```

Comment:
@rcapecce Hi Raul!

I think the this code should be replace by :

`
		if (list.size() == 0)
		{
			list.add(getSupervisor_ID());
		}
`

Is very important that the Supervisor is noticed when a Scheduler is executed successfully and receipt so that one needs to know the process and not have some problems.

Even when this is a report the result of the report is sent to the supervisor as well.

We agreed that it was not necessary to notify the creator if the supervisor was notified and I think we all agreed

Reply:
Hello @rcapecce @niclopup I think that you shoul add this code instead:
```Java
if (list.size() == 0) {
  list.add(getSupervisor_ID());
}
```